test-e2e/panels/filters.test.ts

```diff
@@ -47,7 +47,9 @@ async function baselineCorrectionFilter(
   } else {
     await nmrium.clickTool('baselineCorrection');
   }
-  await nmrium.page.click('button >> text=Apply');
+  await nmrium.page.click('button >> text=Apply', {
```

Comment:
Can you at least add a `// TODO: explanation` comment? We shouldn't push workarounds without minimal documentation.

Reply:
added, thank you